Makefile

```diff
@@ -33,7 +34,7 @@ CPPFILES    := $(wildcard $(PROJDIR)/*.cpp)
 OBJS        := $(patsubst $(PROJDIR)/%.c, $(INTDIR)/%.o, $(CFILES)) $(patsubst $(PROJDIR)/%.cpp, $(INTDIR)/%.o, $(CPPFILES))
 
 # Define final C/C++ flags
-CFLAGS      := -cc1 -triple x86_64-pc-freebsd-elf -munwind-tables -fuse-init-array -debug-info-kind=limited -debugger-tuning=gdb -emit-obj $(EXTRAFLAGS) -isysroot $(TOOLCHAIN) -isystem $(TOOLCHAIN)/include -Iinclude
+CFLAGS      := -cc1 -triple x86_64-pc-freebsd-elf -munwind-tables -debug-info-kind=limited -debugger-tuning=gdb -emit-obj $(EXTRAFLAGS) -isysroot $(TOOLCHAIN) -isystem $(TOOLCHAIN)/include -Iinclude
```

Comment:
If we wish to keep compatibility with clang <~10 this should be put behind an if condition.

Reply:
With the new Open Orbis 0.52 there's no need to use an old version of clang anymore, so I'll run some tests when I have a chance and update all the Makefiles `CFLAGS` to the latest clang.